### ISSUE TITLE
PreparedSQLSniff: False-positive for `array_fill()` with `\count()`

### ISSUE THREAD

**Issue by ocean90** [WordPress/WordPress-Coding-Standards#1875]
## Bug Description
When `count()` is prefixed with the root namespace `\` in the pattern `implode( ',', array_fill( 0, count( $something ), '%s' ) )` you'll get a false-positive for a not prepared query.

## Minimal Code Snippet
```php
$wpdb->prepare(
	"SELECT * FROM $wpdb->something WHERE id IN(" . implode( ',', array_fill( 0, \count( $ids ), '%d' ) ) . ')',
	$ids
);
```

## Error Code
`Use placeholders and $wpdb->prepare(); found \ (WordPress.DB.PreparedSQL.NotPrepared)`

## Environment

| Question               | Answer
| ------------------------| -------
| PHP version             | 7.3.16
| PHP_CodeSniffer version | 7.3.16
| WPCS version            | 2.2.1
| WPCS install type       | Composer global
| IDE (if relevant)       | VS Code

**Comment by jrfnl** [comment]
@ocean90 Thanks for reporting this. I've pulled a quick fix for now - see PR #1876 - though this sniff will need revisiting for proper handling of function calls in the age of namespaces in the future.
